samples/snippets/noxfile.py

```diff
@@ -37,25 +39,31 @@
 
 TEST_CONFIG = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7"],
+    'ignored_versions': [],
+
     # Old samples are opted out of enforcing Python type hints
     # All new samples should feature them
-    "enforce_type_hints": False,
+    'enforce_type_hints': False,
+
     # An envvar key for determining the project id to use. Change it
     # to 'BUILD_SPECIFIC_GCLOUD_PROJECT' if you want to opt in using a
     # build specific Cloud project. You can also use your own string
     # to use your own Cloud project.
-    "gcloud_project_env": "GOOGLE_CLOUD_PROJECT",
+    'gcloud_project_env': 'GOOGLE_CLOUD_PROJECT',
```

Comment:
Why are we not running `black` on this file?

Reply:
Good catch! This will be fixed in https://github.com/googleapis/synthtool/pull/1184.

Reply:
The issue is tracked here https://github.com/googleapis/synthtool/issues/1044